x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/rules/read_rules_route.ts

```diff
@@ -42,7 +45,17 @@ export const createReadRulesRoute: Hapi.ServerRoute = {
         ruleId,
       });
       if (rule != null) {
-        return transformOrError(rule);
+        const ruleStatuses = await savedObjectsClient.find<
+          IRuleSavedAttributesSavedObjectAttributes
+        >({
+          type: ruleStatusSavedObjectType,
+          perPage: 1,
+          sortField: 'statusDate',
+          sortOrder: 'desc',
+          search: rule.id,
+          searchFields: ['alertId'],
+        });
+        return transformOrError(rule, ruleStatuses); // update this to run with an array of rule statuses
```

Comment:
You sure you don't want this to be:

```ts
return transformOrError(rule, ruleStatuses.saved_objects[0]); // update this to run with an array of rule statuses
```

Like in your other call?

Reply:
I probably should update this for clarity however the updates I made to the transformOrError function handle both cases.

Reply:
updated to match other use instances.